tests/test_basic.py

```diff
@@ -0,0 +1,8 @@
+"""Basic tests to verify CI setup."""
+
+
+def test_import() -> None:
+    """Test that the package can be imported."""
+    import microcalibrate
+
+    assert microcalibrate is not None
```

Comment:
Think this is unneeded- it we can't import it, we'll get the error on the previous line.